examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -25,10 +25,14 @@
   },
```

Comment:
<p>Gabriel renamed the notebook to <a href="https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb" target="_blank">02-Merlin-Models-and-NVTabular-integration.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
@bschifferer These two notebooks don't look the same; I think they're different things.

Reply:
In [01-Building-Recommender-Systems-with-Merlin.ipynb](https://github.com/NVIDIA-Merlin/Merlin/pull/186/files/bc57ba599c84a4e3abc1112915a9756d4e005327#diff-c78bb349400fa13bcfb9de5457e74bd4648556f3e0a81621506f5c0c6648025b) we reference a notebook in Merlin Models ( [02-Merlin-Models-and-NVTabular-integration.ipynb](https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb) ).01-Building-Recommender-Systems-with-Merlin.ipynb uses the old filename and old url. We need to update the reference to the new filename/url

Reply:
done.